src/Nethermind/Nethermind.TxPool/TxPool.cs

```diff
@@ -550,5 +630,69 @@ private static void AddNodeInfoEntryForTxPool()
                 $"{(LruCache<KeccakKey, object>.CalculateMemorySize(32, MemoryAllowance.TxHashCacheSize) + LruCache<Keccak, Transaction>.CalculateMemorySize(4096, MemoryAllowance.MemPoolSize)) / 1000 / 1000}MB"
                     .PadLeft(8));
         }
+
+        private void TimerOnElapsed(object? sender, EventArgs e)
+        {
+            WriteTxnPoolReport(_logger);
+
+            _timer!.Enabled = true;
+        }
+
+        private static void WriteTxnPoolReport(ILogger logger)
+        {
+            if (!logger.IsInfo)
+            {
+                return;
+            }
```

Comment:
This shouldn't be Info, Debug at max

Reply:
Is an at a glance summary report like Rpc report, peers report; and it gives insight into the state of the mem pool. 

Have you switched on debug logging? 😱
There's no way you can read anything when logging is debug but would have to go through it in a file